src/util/Log.h

```diff
@@ -111,15 +115,11 @@ class Log {
   static void imbue(const std::locale& locale) { std::cout.imbue(locale); }
 
   static string getTimeStamp() {
-    using namespace std::chrono;
-    auto now = system_clock::now();
-    auto inTimeT = system_clock::to_time_t(now);
-    auto ms = duration_cast<milliseconds>(now.time_since_epoch()) % 1000;
-    std::stringstream ss;
-
-    ss << std::put_time(std::localtime(&inTimeT), "%Y-%m-%d %X") << '.'
-       << std::setw(3) << std::setfill('0') << ms.count();
-    return std::move(ss).str();
```

Comment:
With this change I believe you can remove the includes for `iomanip` and `ctime`, unless I'm missing something